packages/vm/tests/api/state/stateManager.js

```diff
@@ -160,47 +161,35 @@ tape('StateManager', (t) => {
       st.end()
     },
   )
+  t.test('should generate the genesis state root correctly for mainnet from ethereum/tests data', async st => {
+    const genesisData = require('ethereumjs-testing').getSingleFile(
+      'BasicTests/genesishashestest.json',
+    )
+    const stateManager = new StateManager()
+
+    await stateManager.generateCanonicalGenesis()
+    let stateRoot = await stateManager.getStateRoot()
+    st.equals(
+      stateRoot.toString('hex'),
+      genesisData.genesis_state_root,
+      'generateCanonicalGenesis should produce correct state root for mainnet from ethereum/tests data',
+    )
+    st.end()
+  })
 
-  t.test('should generate the genesis state root correctly for mainnet', async (st) => {
-    if (isRunningInKarma()) {
```

Comment:
I am a bit confused to see an `async` refactor in this PR. Was it ported from another branch, or the changes were made here indeed? If the latter, it becomes harder to review when there are changes from  different scopes.

Also, is this test still being skipped on browsers?

Reply:
This is not changed here but is probably the result of a rebase which went wrong. I don't know how to fix it, but for some reason at one of the rebases it shows all changes from master as newly added/deleted items 😢